File: Year 2/Semester 2/Web/LAB7-Angular+PHP/my-app/src/app/Document.ts
export interface Document{
  id: number;
  author: string;
  title: string;
  pages: number;
  types: string;
  format: string;
}

export interface AddDocumentDto{
  author: string;
  title: string;
  pages: number;
  types: string;
  format: string;
}

// https://devooti.com/angular-crud-operations-with-php-and-mysql/

File: Year 2/Semester 2/Web/LAB9 - ASP.NET/WebApplication1/WebApplication1/Models/Document.cs
namespace WebApplication1.Models;

public class Document
{
    public int id { get; set; }
    
    public String author { get; set; }
    
    public String title { get; set; }
    
    public int pages { get; set; }
    
    public String types { get; set; }
    
    public String format { get; set; }

    public Document()
    {
        author = "";
        title = "";
        pages = 0;
        types = "";
        format = "";
    }
}